client/asset/btc/electrum.go

```diff
@@ -118,11 +119,8 @@ func (btc *ExchangeWalletElectrum) Connect(ctx context.Context) (*sync.WaitGroup
 		return nil, err
 	}
 	var hasFreezeUTXO bool
-	for i := range commands {
-		if commands[i] == "freeze_utxo" {
-			hasFreezeUTXO = true
-			break
-		}
+	if slices.Contains(commands, "freeze_utxo") {
+		hasFreezeUTXO = true
```

Comment:
Sorry, looking again this can be simplified more, including line 121

```suggestion
        hasFreezeUTXO := slices.Contains(commands, "freeze_utxo")
```